src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,31 @@
+package duke.command;
+
+import duke.main.Storage;
+import duke.main.TaskList;
+import duke.main.Ui;
+
+/** Command to find tasks based on search string */
+public class FindCommand implements Command {
+    private final String searchString;
```

Comment:
searchString sounds like a verb, maybe can name it to something like stringToSearch? (although searchString still sounds better)

Reply:
hmm yeah, it sounds a bit odd, but I will change it to stringToSearch just to be safe :D